mediator/med_io_mod.F90

```diff
@@ -900,15 +908,27 @@ subroutine med_io_write_FB(filename, FB, whead, wdata, nx, ny, nt, &
     ! all the global grid values in the distgrid - e.g. CTSM
 
     ng = maxval(maxIndexPTile)
-    lnx = ng
-    lny = 1
+    if (atmtiles) then
+      lnx = tilesize
+      lny = tilesize
+      ntiles = ng/(lnx*lny)
+      write(tmpstr,*) subname, 'ng,lnx,lny,ntiles = ',ng,lnx,lny,ntiles
+      call ESMF_LogWrite(trim(tmpstr), ESMF_LOGMSG_INFO)
+      if (ntiles /= 6) then
```

Comment:
So at this time, we can only write global fv3 files on tiles, not regional fv3 which has one tile?

Reply:
Yes, that is right. I did not attempt to make the CMEPS change for anything other than global. Other than HAFS I don't know of another application which uses both FV3atm and CMEPS and I wasn't sure enough about the various configurations HAFS runs.